src/main/java/org/openrewrite/java/template/RefasterTemplateProcessor.java

```diff
@@ -301,18 +310,26 @@ public void visitClassDef(JCTree.JCClassDecl classDecl) {
                                 .map(Map.Entry::getValue)
                                 .flatMap(Set::stream)
                                 .collect(Collectors.toSet());
+                        Set<String> usedMethods = new LinkedHashSet<>();
                         for (String import_ : staticImports.values().stream().flatMap(Set::stream).collect(Collectors.toSet())) {
                             if (import_.startsWith("java.lang.")) {
                                 continue;
```

Comment:
We miss out on some used methods here.

Reply:
Good call; picked up in https://github.com/openrewrite/rewrite-templating/pull/24/commits/34e9ba9df053ff2ef5b81b8e9b35f90bb06da8ec